stacker/exceptions.py

```diff
@@ -214,7 +214,6 @@ def __init__(self, failed_stacks, *args, **kwargs):
         self.failed_stacks = failed_stacks
 
         stack_names = ', '.join(stack.name for stack in failed_stacks)
-        message = ("The following stacks failed their plans: %s\n"
-                   % (stack_names,))
+        message = ("The following stacks failed: %s\n" % (stack_names,))
```

Comment:
minor nit: Shouldn't need the outer parenthesis on this, since it doesn't cross 80 characters.

Reply:
Fixed :)